tests/rolling.test.js

```diff
@@ -399,6 +399,34 @@ describe('Rolling', () => {
       // remove the spy
       spy.mockRestore();
     });
+
+    test('roll `(d6>3)?1:0', () => {
```

Comment:
Again, looking good. Would you also be able to add tests to ensure that it correctly handles more advanced expressions as well?